hibernate-core/src/main/java/org/hibernate/dialect/identity/DB2zIdentityColumnSupport.java

```diff
@@ -9,7 +9,7 @@
 /**
  * @author Andrea Boriero
  */
-public class DB2390IdentityColumnSupport extends DB2IdentityColumnSupport {
+public class DB2zIdentityColumnSupport extends DB2IdentityColumnSupport {
```

Comment:
Nice change, but please introduce a `DB2390IdentityColumnSupport` class that now extends `DB2zIdentityColumnSupport` and is `@Deprecated`, to retain backwards compatibility.

Reply:
Hadn't seen your previous comment, do you want this in the community-dialects project, or in core?

Reply:
In core